app/banner.js

```diff
@@ -6,6 +6,11 @@ module.exports = function (app) {
   // Detect if banner should be shown based on cookies set
   app.use(cookieParser())
   app.use(function (request, response, next) {
+    if ('hide_banner' in request.query) {
```

Comment:
would prefer kebab case but doesnt matter tbf

Reply:
Should it set a cookie so you dont require it for subsequent navigations?

Reply:
Re cookies, I don't need that functionality for my use case (Browserstack) so reckon we keep it simple for now.